README.md

```diff
@@ -52,3 +52,6 @@ ROS2-approved formatters are included in the IDE.
 
 * **c++** uncrustify; config from `ament_uncrustify`
 * **python** autopep8; vscode settings consistent with the [style guide](https://index.ros.org/doc/ros2/Contributing/Code-Style-Language-Versions/)
+
+## Notes on testing pdf_beamtime_server
```

Comment:
Explain below what pdf_beamtime_server is: 
i.e. a WIP for deploying the ur at the pdf beamline. Add some detail to help the re-direct.

Reply:
Added description on the package and the pointer.